drafts/2020-05-12-this-week-in-rust.md

```diff
@@ -24,8 +26,24 @@ If you find any errors in this week's issue, [please submit a PR](https://github
 * [time_it: A case study in Rust macros](https://notes.iveselov.info/programming/time_it-a-case-study-in-rust-macros)
 * [Dynamic stylesheets and Yew](https://conradludgate.com/posts/yew-css/)
 * [A practical introduction to async programming in Rust](http://jamesmcm.github.io/blog/2020/05/06/a-practical-introduction-to-async-programming-in-rust/#en)
-
-+ [Collecting many errors from an iterator of Results](https://tarquin-the-brave.github.io/blog/posts/collecting-all-the-errors/).
+* [Collecting many errors from an iterator of Results](https://tarquin-the-brave.github.io/blog/posts/collecting-all-the-errors/).
+* [Auto-currying Rust Functions](https://peppe.rs/posts/auto-currying_rust_functions/)
+* [Converting bits to integers in Rust using generics](https://dev.to/citizen_stig/converting-bits-to-integers-in-rust-using-generics-2nfg)
+* [Magnifying Glasses for Rust Assembly](https://www.justanotherdot.com/posts/magnifying-glasses-for-rust-assembly.html)
+* [More Rust and Load Balancer Adventures](https://medium.com/@bparli/more-rust-and-load-balancer-adventures-fad07f4fb095) 
+* [Porting North Korean Dictionaries with Rust](https://digitalnk.com/blog/2020/05/08/porting-north-korean-dictionaries-with-rust/)
+* [Rust verification tools](https://alastairreid.github.io/rust-verification-tools/)
+* [Series Announcement - Zero to Production in Rust](https://www.lpalmieri.com/posts/2020-05-10-announcement-zero-to-production-in-rust/)
+* [Super Simple Disk Benchmark written in rust](https://www.d34dl0ck.me/super-simple-disk-benchmark-written-in-rust.html#super-simple-disk-benchmark-written-in-rust)
```

Comment:
This one is a crate announcement with not a lot of content about HOW it does what it does in Rust or how it is implemented. Removing this one.